### PR TITLE
feat(kit): `InputPhoneInternational` allow paste/drop without code

### LINKED ISSUE #1571: 🐞 - `InputPhoneInternational` doesn't paste/drop without code
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/taiga-starter

### Description

If we paste number without code, we should paste data with current code

### Angular version

9>

### Taiga UI version

2.39.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?
Closes #1571 

## What is the new behavior?
paste value with current code if no code found from value

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts [hunk 1]
```diff
@@ -205,40 +205,41 @@ describe('InputPhoneInternational', () => {
             expect(testComponent.control.value).toEqual('+380123456789');
         });
 
-        it('should not update value on paste if there is no country code in list', () => {
+        it('should set country code on paste event 8(863)', () => {
```
Comment (splincode) [inline projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts:208]: Why was it necessary to rewrite the tests, and not add new ones?

Comment (splincode) [reply projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts:208]: sounds like a `feat` ))

**Review by splincode — APPROVED** [review]

File: projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts:208]: ok I'l rename it :)

File: projects/core/constants/regexp.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ export const TUI_DIGIT_REGEXP = /\d/;
 export const TUI_NON_DIGIT_REGEXP = /\D/;
 export const TUI_NON_DIGITS_REGEXP = /\D+/g;
 export const TUI_LEADING_ZEROES_REGEXP = /^0+/;
+export const TUI_NON_PLUS_AND_DIGITS_REGEX = /[ \-_()]/g;
```
Comment (nsbarsukov) [inline projects/core/constants/regexp.ts:5]:
```suggestion
export const TUI_NON_PLUS_AND_DIGITS_REGEX = /[^\d\+]/g;
```

Comment (nsbarsukov) [reply projects/core/constants/regexp.ts:5]: everything not digits and not plus

Comment (nsbarsukov) [reply projects/core/constants/regexp.ts:5]: P.S. i am not master of reg exps.
Please double check me :)

Comment (splincode) [reply projects/core/constants/regexp.ts:5]: image

Comment (vladimirpotekhin) [reply projects/core/constants/regexp.ts:5]: ok, I think naming is wrong. I moved regexp from input-phone, it uses to clear value from brackets, dashes and underscores. How about TUI_MASK_SYMBOLS_REGEX?

**Pull request merged**